src/Acuminator/Acuminator.Vsix/Resources/VSIXResource.resx

```diff
@@ -240,6 +240,9 @@
   <data name="CodeMap_ExtraInfo_PXSetupViewIconTooltip" xml:space="preserve">
     <value>This view is PXSetup or derived from it</value>
   </data>
+  <data name="CodeMap_ExtraInfo_ProcessingGraphExtensionIconTooltip" xml:space="preserve">
```

Comment:
@EAndrosova please review the new tooltip message for a processing icon indicator for graph extensions.